package.json

```diff
@@ -59,7 +59,7 @@
   "homepage": "https://github.com/brave-intl/bat-ledger",
   "dependencies": {
     "@octokit/rest": "^15.5.1",
-    "@uphold/uphold-sdk-javascript": "^2.0.4",
+    "@uphold/uphold-sdk-javascript": "github:brave-intl/uphold-sdk-javascript",
```

Comment:
this might be extra effort to maintain our fork.
our plan might be to switch back, and in that case an issue and comment explaining this would be nice

Reply:
opened #258 to track